### PR TITLE
Simplepie is removed since we don't use it anymore

### PR DESCRIPTION
**Pull request by annando**
We had used that library in the DFRN import and in the probe_url function. Now we don't use it anymore.

### CODE REVIEW

**Comment by tobiasd** [comment]
There is no other use of the library in core or addons?

**Comment by annando** [comment]
The only other place was the Scrape.php. I did a grep for the relevant parts.

**Comment by annando** [comment]
But please wait with the pull request. Today I had the feeling that there might be a possible side effect that I haven't checked.

File: include/feed.php [hunk 1]
```diff
@@ -2,7 +2,7 @@
 require_once("include/html2bbcode.php");
 require_once("include/items.php");
 
-function feed_import($xml,$importer,&$contact, &$hub) {
+function feed_import($xml,$importer,&$contact, &$hub, $simulate = false) {
```
Comment (fabrixxm) [inline include/feed.php]: Could you add some doxygen comment? :-)

**Comment by annando** [comment]
I added a documentation. Additionally I fixed the problem that I described above.

**Comment by fabrixxm** [comment]
have you anything else to do here? can we merge?

**Comment by annando** [comment]
I'm done. From my point of view it can be merged.

**Pull request merged**